server/model/models.go

```diff
@@ -0,0 +1,54 @@
+package model
+
+type User struct {
```

Comment:
we defined this model in schema task, but it doesn't have all necessary field. For example avatar url.
I would also like to abstract from github in model. `login` and `username` fields are better in my opinion. Then we can add other providers later. Google for example. Or allow registering user. If you use abstract model consumers of model don't need to know how user was created.
Please share your thought!

Reply:
I just tried to follow our [specs](https://github.com/src-d/code-annotation/issues/2#issuecomment-357315069) because I was told that other log-in providers are not expected in the middle term. (We even considered using the `gh-user-name` as the `User.ID`) Anyhow, I'd consider https://github.com/src-d/code-annotation/pull/22/files#diff-c318b873cebb799af6f409abe8089672 for sure.

About storing the avatar... I'd like to confirm it with @src-d/product, because in our former web apps they requested us to do not store it, but loading it from gh (so it would be always updated) `htps://avatars2.githubusercontent.com/u/{gh-user-id}`

To do so, let us know if you can retrieve the `gh-user-id` from GH, or if we should store it in the DB.

Reply:
Github return `https://avatars0.githubusercontent.com/u/406916?v=4` as avatar url. I suggest to store it.

Reply:
It's a business decision.

@bzz should we keep the user avatar in sync with the real one?
- **in sync**: showing whatever avatar GH shows under `htps://avatars2.githubusercontent.com/u/{gh-user-id}`
- **outdated**: showing a stored version when the user first logged in

Reply:
It's totally fine, either not to have avatar at all, or to show an outdated on.

Reply:
406916 is a userId, so the image itself is always fresh.